setup.py

```diff
@@ -178,6 +241,6 @@ def is_editable_install_(self):
     include_package_data=True,
     long_description_content_type="text/markdown",
     ext_modules=ext_modules,
-    cmdclass=dict(build_ext=BUDAEagerBuild),
```

Comment:
can we delete `BUDAEagerBuild` now?

Reply:
if we want to leave the `make` flow around, it'll need BUDAEagerBuild